[2023-12-15T22:48:52.845+0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag2.load_data_to_github manual__2023-12-15T17:48:29.968051+00:00 [queued]>
[2023-12-15T22:48:52.852+0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag2.load_data_to_github manual__2023-12-15T17:48:29.968051+00:00 [queued]>
[2023-12-15T22:48:52.853+0500] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-15T22:48:53.008+0500] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data_to_github> on 2023-12-15 17:48:29.968051+00:00
[2023-12-15T22:48:53.011+0500] {standard_task_runner.py:57} INFO - Started process 12107 to run task
[2023-12-15T22:48:53.015+0500] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dag2', 'load_data_to_github', 'manual__2023-12-15T17:48:29.968051+00:00', '--job-id', '1359', '--raw', '--subdir', 'DAGS_FOLDER/airflow_dag.py', '--cfg-path', '/tmp/tmps9onu8cx']
[2023-12-15T22:48:53.016+0500] {standard_task_runner.py:85} INFO - Job 1359: Subtask load_data_to_github
[2023-12-15T22:48:53.146+0500] {task_command.py:416} INFO - Running <TaskInstance: etl_dag2.load_data_to_github manual__2023-12-15T17:48:29.968051+00:00 [running]> on host amina-HP-ProBook-450-G2
[2023-12-15T22:48:53.512+0500] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='amina_bibi' AIRFLOW_CTX_DAG_ID='etl_dag2' AIRFLOW_CTX_TASK_ID='load_data_to_github' AIRFLOW_CTX_EXECUTION_DATE='2023-12-15T17:48:29.968051+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-15T17:48:29.968051+00:00'
[2023-12-15T22:49:07.152+0500] {python.py:194} INFO - Done. Returned value was: None
[2023-12-15T22:49:07.156+0500] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_dag2, task_id=load_data_to_github, execution_date=20231215T174829, start_date=20231215T174852, end_date=20231215T174907
[2023-12-15T22:49:07.403+0500] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-15T22:49:07.419+0500] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
